docs/v1/P4Runtime-Spec.mdk

```diff
@@ -767,10 +767,9 @@ from making changes or even reading the state of the server.
 ## Default Role
 
 A controller can omit the role message in `MasterArbitrationUpdate`. This
-implies the "default role", which corresponds to "full pipeline access". This
-also implies that a default role has a `role.id` of 0 (default). If using a
-default role, all RPCs from the controller (&eg; `Write`) must set the `role_id`
-to 0.
+implies the "default role", which corresponds to "full pipeline access".
+If using a default role, all RPCs from the controller (&eg; `Write`) must
```

Comment:
Should we mention explicitly that the empty string is therefore reserved for the default role and that it is not a valid name for a user-defined role? I noticed that you remove the sentence about role id 0.

Reply:
I'm pretty sure that even in protobuf v3, empty string "" is different from unset string. Therefore, I think "" need not be a special case, and can just be like any other role.

Reply:
No, string is a scalar field and empty string is the default
https://developers.google.com/protocol-buffers/docs/proto3#default

Reply:
Oh you are right, thanks! I though strings were different.